src/de/map.rs

```diff
@@ -74,6 +74,10 @@ impl<'a, 'de, R: BufRead> de::MapAccess<'de> for MapAccess<'a, R> {
                     self.value = MapValue::InnerValue;
                     seed.deserialize(INNER_VALUE.into_deserializer()).map(Some)
                 }
+                Some(Event::CData(e)) => {
```

Comment:
```suggestion
                Some(Event::CData(_)) => {
```
I don't see where e is used